politeiad/backend/tlogbe/tlogclient.go

```diff
@@ -129,6 +144,18 @@ func (c *backendClient) blobsByMerkle(tlogID string, token []byte, merkles [][]b
 		return nil, err
 	}
 
+	if len(token) == tokenPrefixSize() {
```

Comment:
Why do you need this here is you've already implemented it in `treeIDFromToken()`?

Reply:
no need